wiki/Glossary/en.md

```diff
@@ -474,54 +459,55 @@ Thus mappers are advised to use it judiciously.
 
 ### Kudosu!
```

Comment:
The kudosu image has the description and the tooltip and the kudosu button is the same text.